src/BenchmarkDotNet.Core/Portability/HyperV.cs

```diff
@@ -0,0 +1,13 @@
+namespace BenchmarkDotNet.Portability
+{
+    public class HyperV : VirtualMachineHypervisor
+    {
+        public override string Name => "Hyper-V";
+
+        public override bool IsVirtualMachine(string manufacturer, string model)
+        {
+            return manufacturer != null && model != null && 
```

Comment:
Sometimes null is valid for VM detector. For example, `VMware` does not care about `manufacturer` value.

To avoid so many manual null checks, I can provide base method like this one
```csharp
protected bool ContainsVmIdentifier(string systemInformation, string vmIdentifier)
{
    return systemInformation != null && systemInformation.ContainsWithIgnoreCase(vmIdentifier);
}
```

with following usage in VMware:
```csharp
public override bool IsVirtualMachine(string manufacturer, string model)
{
    return ContainsVmIdentifier(model, "vmware");
}
```
What do you think @AndreyAkinshin ?